README.md

```diff
@@ -45,10 +53,34 @@ Run the operator:
 $ make run
 ```
 
+Clean up the Kubernetes/OpenShift cluster:
+```
+$ make cluster/clean
+```
+
 ## Via the Operator Catalog
 
 ***In development***
 
+## Usage
+### Provider
+A config map object is expected to exist with a mapping from type name to deployment method, and example of this can be seen [here](deploy/examples/cloud_resource_config.yaml)
```

Comment:
Would a mapping from resource type to desired provider be more clear? 

```suggestion
A config map object is expected to exist with a mapping from type name to deployment method, an example of this can be seen [here](deploy/examples/cloud_resource_config.yaml).
```